src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -246,14 +246,14 @@ def random_integers(
 def filter_by_rate(
     sims_to_filter: Union[pd.Index, pd.Series],
```

Comment:
per comment in other PR (this is not the correct place, but since you're touching the function in this PR anyway, maybe this is an okay place to tweak this without opening a whole new PR) instead of `sims_to_filter`, maybe something like `entities_to_filter`? given that it could be household_ids or business_ids (i think, right?)